plugin/secrets_service_account_key.go

```diff
@@ -174,26 +174,17 @@ func (b *backend) secretKeyRevoke(ctx context.Context, req *logical.Request, d *
 }
 
 func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleSet, keyType, keyAlgorithm string, ttl int) (*logical.Response, error) {
-	cfg, err := getConfig(ctx, s)
```

Comment:
removing unused cfg, it is left over from when we used to look up TTL from config